src/csharp/Extensions/README.md

```diff
@@ -0,0 +1,13 @@
+# .Net for Spark Extensions
+
+## Table of Contents
```

Comment:
The ReadMe is very short.  Do we need the table of contents ?

Reply:
I added it for future proofing.